pyblog/articles/models.py

```diff
@@ -0,0 +1,17 @@
+import datetime
+
+from django.db import models
+from django.utils import timezone
+from django.utils.encoding import python_2_unicode_compatible
+
+# Create your models here.
+@python_2_unicode_compatible  # only if you need to support Python 2
+class Comment(models.Model):
+    comment_text = models.CharField(max_length=200)
```

Comment:
It's missing the atuthor fields - name and email, and also information about when the comment was created (and updated)